pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -192,36 +200,60 @@ func handleErr(reqLogger logr.Logger, status *cv1beta1.CincinnatiStatus, reason
 	reqLogger.Error(e, reason)
 }
 
+// handleCACertStatus logs the message and sets an appropriate Condition on the ConditionRegistryCACertFound status.
+func handleCACertStatus(reqLogger logr.Logger, status *cv1beta1.CincinnatiStatus, reason string, message string) {
+	conditionsv1.SetStatusCondition(&status.Conditions, conditionsv1.Condition{
+		Type:    cv1beta1.ConditionRegistryCACertFound,
+		Status:  corev1.ConditionFalse,
+		Reason:  reason,
+		Message: message,
+	})
+	reqLogger.Info(message)
+}
+
 func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, reqLogger logr.Logger, instance *cv1beta1.Cincinnati, resources *kubeResources) error {
 	// Check if the Cluster is aware of a registry requiring an
 	// AdditionalTrustedCA
 	image := &apicfgv1.Image{}
 	err := r.client.Get(ctx, types.NamespacedName{Name: defaults.ImageConfigName, Namespace: ""}, image)
 	if err != nil && errors.IsNotFound(err) {
+		m := fmt.Sprintf("image.config.openshift.io not found for (Name: %v, Namespace: %v)", defaults.ImageConfigName, "")
+		handleCACertStatus(reqLogger, &instance.Status, "EnsureAdditionalTrustedCAFailed", m)
```

Comment:
In case like this, when cert is not found, the reconcile keeps moving and completes the process. So you get two status conditions:
```
status:
    conditions:
    - lastHeartbeatTime: "2020-05-06T20:32:36Z"
      lastTransitionTime: "2020-05-06T20:32:36Z"
      reason: Success
      status: "True"
      type: ReconcileCompleted
    - lastHeartbeatTime: "2020-05-06T20:32:36Z"
      lastTransitionTime: "2020-05-06T20:32:36Z"
      message: 'image.config.openshift.io.Spec.AdditionalTrustedCA.Name not found
        for image (Name: cluster, Namespace: )'
      reason: EnsureAdditionalTrustedCAFailed
      status: "False"
      type: RegistryCACertFound
```